internal/pkg/runtime/launch/launcher_linux.go

```diff
@@ -866,6 +866,11 @@ func (l *Launcher) setNvCCLIConfig() (err error) {
 	}
 	l.engineConfig.SetNvCCLIEnv(nvCCLIEnv)
 
+	if len(l.cfg.OverlayPaths) > 0 && l.cfg.Writable {
+		sylog.Infof("Detected writable overlay images, skipping setting --writable-tmpfs (required by nvidia-container-cli)")
```

Comment:
Let's make this Verbosef instead of Infof

Reply:
Also, "images" can be misleading, because it should also work with an overlay directory.  So change the message to
```
"Detected writable overlay, skipping setting --writable-tmpfs (otherwise required by --nvccli)"
```

Reply:
thanks! updated as suggested.